packages/instantsearch-ui-components/src/components/__tests__/Hits.test.tsx

```diff
@@ -41,85 +45,27 @@ describe('Hits', () => {
             <li
               class="ais-Hits-item"
             >
-              <div
-                style="word-break: break-all;"
-              >
-                {"objectID":"abc","__position":1}
-                …
-              </div>
+              abc
             </li>
             <li
               class="ais-Hits-item"
             >
-              <div
-                style="word-break: break-all;"
-              >
-                {"objectID":"def","__position":2}
-                …
-              </div>
+              def
             </li>
           </ol>
         </div>
       </div>
     `);
   });
 
-  test('renders with custom hitComponent', () => {
-    const props = createProps({
-      hits: [
-        { objectID: 'abc', __position: 1, something: true },
-        { objectID: 'def', __position: 2, something: false },
-      ],
-      hitComponent: ({ hit }) => (
-        <strong>{`${hit.objectID} - ${hit.something}`}</strong>
-      ),
-    });
-
-    const { container } = render(<Hits {...props} />);
-
-    expect(container.querySelectorAll('strong')).toHaveLength(2);
-
-    expect(container).toMatchInlineSnapshot(`
-      <div>
-        <div
-          class="ais-Hits"
-        >
-          <ol
-            class="ais-Hits-list"
-          >
-            <li
-              class="ais-Hits-item"
-            >
-              <strong>
-                abc - true
-              </strong>
-            </li>
-            <li
-              class="ais-Hits-item"
-            >
-              <strong>
-                def - false
-              </strong>
-            </li>
-          </ol>
-        </div>
-      </div>
-    `);
-  });
-
-  test('passes sendEvent to hitComponent', () => {
+  test('renders with custom div props', () => {
```

Comment:
Just for the sake of consistent test names, no biggie.

```suggestion
  test('forwards `div` props to the root element', () => {
```